media-source-respec.html

```diff
@@ -102,7 +102,7 @@
 
     <link rel="stylesheet" href="mse.css">
   </head>
-  <body data-cite="html dom url fileapi">
+  <body data-cite="html dom url fileapi infra">
     <section id="abstract">
       This specification extends {{HTMLMediaElement}} [[HTML]] to allow
       JavaScript to generate media streams for playback.
```

Comment:
@wolenetz, can you clarify a bit more what you mean?

Reply:
My apologies for imprecise review comment location. See the first <p> in the "sotd" section beginning in line 114. Consider changing it to something like the following: (It might also be a good time to put this list of substantive changes in an ordered list, since it has grown quite long and lacks consistent parallel sentence structure otherwise.)
``
      <p>On top of editorial updates, substantives changes since publication as a W3C Recommendation in <a href="https://www.w3.org/TR/2016/REC-media-source-20161117/">November 2016</a> are the addition of a {{SourceBuffer/changeType()}} method to switch codecs, the possibility to create and use {{MediaSource}} objects off the main thread in dedicated workers, the removal of the <code>createObjectURL()</code> extension to the {{URL}} object following its integration in the File API [[FILEAPI]], and the addition of {{ManagedMediaSource}}, ManagedSourceBuffer}}, and {{BufferedChangeEvent}} interfaces supporting power-efficient streaming and active buffered media cleanup by the user agent. For a full list of changes done since the previous version, see the <a href="https://github.com/w3c/media-source/commits/main">commits</a>.</p>
``

Reply:
Added @wolenetz suggested text to the SOTD.